library/src/lib/calendar/calendar.component.ts

```diff
@@ -65,120 +68,170 @@ let calendarUniqueId: number = 0;
     daysPerMonth = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
 
+    /** @hidden The displayed grid of calendar days. Internal use */
     calendarGrid: CalendarDay[][] = [];
+    /** @hidden The years to display in the year selection grid. Internal use */
     calendarYearsList: number[] = [];
 
+    /** @hidden Today's date. Internal use */
     today: Date = new Date();
+    /** @hidden Today's month. Internal use */
     todayMonth = this.today.getMonth();
+    /** @hidden Today's year. Internal use */
     todayYear = this.today.getFullYear();
 
+    /** @hidden Date used to navigate the calendar. Not to be confused with selectedDay which is the ngModel. Internal use */
     date: Date = new Date();
+    /** @hidden Month used to navigate the calendar.Internal use */
     month: number = this.date.getMonth();
+    /** @hidden Name of the month currently displayed. Internal use */
     monthName: string;
+    /** @hidden Year used to navigate the calendar. Internal use */
     year: number = this.date.getFullYear();
+    /** @hidden Day of month used to navigate the calendar. Internal use */
     day = this.date.getDate();
 
+    /** @hidden Number (0-11) of the selected month used to navigate the calendar. Internal use */
     selectedMonth: number;
+
+    /** @hidden The first year to be displayed in the list of selectable years. Internal use */
     firstYearCalendarList: number = this.year;
+
+    /** @hidden The first year to be displayed in the list of selectable years. Internal use */
     selectCounter: number = 0;
 
+    /** The currently selected CalendarDay model */
     @Input()
     selectedDay: CalendarDay = {
         date: null
     };
+    /** Fired when the selectedDay model changes */
     @Output()
     selectedDayChange = new EventEmitter();
 
+    /** The currently selected first CalendarDay in a range type calendar */
     @Input()
     selectedRangeFirst: CalendarDay = {
         date: null
     };
+    /** Fired when the selectedRangeFirst model changes */
     @Output()
     selectedRangeFirstChange = new EventEmitter();
 
+    /** The currently selected last CalendarDay in a range type calendar */
     @Input()
     selectedRangeLast: CalendarDay = {
         date: null
     };
+    /** Fired when the selectedRangeLast model changes */
     @Output()
     selectedRangeLastChange = new EventEmitter();
 
+    /** @hidden The date that gets emitted to the datePicker when the select day changes on the calendar. Internal use */
     emittedDate: EmittedDate = {
         selectedDay: this.selectedDay,
         selectedFirstDay: this.selectedRangeFirst,
         selectedLastDay: this.selectedRangeLast
     };
 
+    /** Fired when the calendar is closed */
     @Output()
     closeCalendar = new EventEmitter<any>();
 
+    /** @hidden Subscription to the i18n service */
     private i18nLocalSub: Subscription;
 
+    /** @param d Function used to disable certain dates in the calendar. */
```

Comment:
you are setting the description of the function **parameter** to `Function used to disable certain dates in the calendar.` here.

Description of the function goes above param description, see: https://github.com/SAP/fundamental-ngx/wiki/Documenting-Code#functions